### DISCUSSION TITLE
Upgrade transformer-pug dependencies to fix vulnerability

### DISCUSSION THREAD

**Discussion by dragoonzx** [parcel-bundler/parcel#6104] [Ideas] [answered]
NPM audit security report that we have high severity vulnerability. Now transformer using pug@2.0.4, can I patch it to pug@3.0.1?

Upgrade should solve the issue

**Comment by mischnic** [comment, marked as answer]
I don't see why not. Pug 3 has been out for almost a year, and these breaking changes should happen before Parcel 2 hits stable

**Reply by dragoonzx** [reply]
Good, PR: https://github.com/parcel-bundler/parcel/pull/6108
